lib/auth/auth_test.go

```diff
@@ -76,6 +76,7 @@ type AuthSuite struct {
 var _ = Suite(&AuthSuite{})
 
 func (s *AuthSuite) SetUpTest(c *C) {
+	ctx := context.Background()
```

Comment:
Since you're adding a `ctx` variable here, could you replace all the `context.TODO` occurrences in this function with `ctx` for the sake of consistency?